Evaluation/Input/Content/References.md

```diff
@@ -2,7 +2,7 @@
 
 **Baneyx 2014** Baneyx, G., Parrott, N., Meille, C., Iliadis, A. & Lavé, T. Physiologically based pharmacokinetic modeling of CYP3A4 induction by rifampicin in human: influence of time between substrate and inducer administration. Eur. J. Pharm. Sci. 56, 1–15 (2014).
 
-**DrugBank DB00802** https://www.drugbank.ca/drugs/DB00802, accessed 05-15-2020.
+**DrugBank DB00802** [https://www.drugbank.ca/drugs/DB00802](https://www.drugbank.ca/drugs/DB00802), accessed 05-15-2020.
```

Comment:
❗ 
Always better to use the markdown link formatting instead of putting the url directly in text
(e.g. when just putting as text: the hyperlink is not created in word/pdf report and not properly checked by some url checkers)